spinalcordtoolbox/scripts/sct_testing.py

```diff
@@ -0,0 +1,19 @@
+"""
+A wrapper for pytest to provide backwards compatability for `sct_testing`.
+
+Context: SCT previously provided its own custom testing framework using a
+         script called `sct_testing`. This was replaced with `pytest`, however
+         sct_testing is likely still remembered by users, and may still be in
+         use in scripts. So, this script provides a way to use `sct_testing`
+         to run the pytest test suite.
+"""
+
+import pytest
+import sys
+
+from spinalcordtoolbox import __sct_dir__
+
+if __name__ == "__main__":
+    # Treat `sct_testing` as an alias to `pytest`. If no arguments are passed,
+    # simply point pytest at the repo (to avoid looking for tests in the pwd).
+    sys.exit(pytest.main(sys.argv[1:] if sys.argv[1:] else [__sct_dir__]))
```

Comment:
Do we want a proper argparse interface for this?

Right now, the way it works is this:

* Running `sct_testing` with no args will run all the tests.
* Running `sct_testing` with arguments will just pass those args to `pytest`.

My thinking is that in 99% of cases, our users will just run `sct_testing` to run all of the tests. 

But, our old `sct_testing` command had options like `-function sct_deepseg`, etc. I could replicate that functionality too, but it would make this PR a little more complicated.

Reply:
i agree, no strong need for argparse